### PR TITLE
Add usage example to math_ops.py

### PR DESCRIPTION
**Pull request by RichardXiao13**

### CODE REVIEW

File: tensorflow/python/ops/math_ops.py [hunk 1]
```diff
@@ -4110,6 +4110,15 @@ def polyval(coeffs, x, name=None):
     A `tensor` of the shape as the expression p(x) with usual broadcasting rules
     for element-wise addition and multiplication applied.
 
+  Usage Example:
+  '''python
```
Comment (kyscg) [inline tensorflow/python/ops/math_ops.py]:
```suggestion
  ```python
```

File: tensorflow/python/ops/math_ops.py [hunk 2]
```diff
@@ -4110,6 +4110,15 @@ def polyval(coeffs, x, name=None):
...
+    import tensorflow as tf
```
Comment (kyscg) [inline tensorflow/python/ops/math_ops.py]:
```suggestion
   
```

File: tensorflow/python/ops/math_ops.py [hunk 3]
```diff
@@ -4110,6 +4110,15 @@ def polyval(coeffs, x, name=None):
...
+    coefficients = [1.0, 2.5, -4.2]
+    x = 5.0
+    y = tf.math.polyval(coefficients, x)
+    # y = 33.3
+  '''
```
Comment (kyscg) [inline tensorflow/python/ops/math_ops.py]:
```suggestion
  ```
```

**Review by kyscg — CHANGES_REQUESTED** [review]

**Comment by rthadur** [comment]
Closing this PR as this not against `master`, please open a new PR against `master`

**Pull request closed without merging**